CHANGELOG

```diff
@@ -1,3 +1,10 @@
+v3.4.6 (2019-XX-XX)
```

Comment:
thanks for creating the changelog.  to be double checked before merging. I believe this will go in 3.4.5 (not 3.4.6). Leaving this to the assigned reviewer.

Reply:
👍